setup_process_exporter.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Setup Process Exporter
+  hosts: all
+  become: true
+  gather_facts: true
+  vars:
+    node_service_identifier: "{{ group_names[0] }}{{ '' if type in default_node_types else '-' + type }}"
```

Comment:
we can remove {{ '' if type in default_node_types else '-' + type }} so that its just chainflip-process-exporter and not chainflip-node-process-exporter or chainflip-engine-process-exporter